lib/Ohmbrewer_Screen.cpp

```diff
@@ -189,7 +211,9 @@ unsigned long Ohmbrewer::Screen::displayRelays() {
             ((Ohmbrewer::Relay*)(*itr))->display(this);
         }
     }
-    printMargin(2);
+    if(foundFirst) {
+        printMargin(2);
```

Comment:
Was just making it consistent with the other ones. They all do this and just looking at Relay::doDisplay vs, for example, TemperatureSensor::doDisplay they end the same way breaks-wise (a println at the end).